packages/flame/test/effects2/transform2d_effect_test.dart

```diff
@@ -8,9 +8,6 @@ import 'package:flutter_test/flutter_test.dart';
 
 class MyEffect extends Transform2DEffect {
```

Comment:
Is `Transform2DEffect` abstract? Otherwise that could just be used directly.

Reply:
Yes, it's abstract because in itself it doesn't do anything (other than keeping track of `lastProgress` variable). So I don't see how this do-nothing behavior would be useful anywhere, other than in this particular test file.